package.json

```diff
@@ -42,6 +42,29 @@
     ],
     "main": "./out/extension",
     "contributes": {
+        "languages": [
+            {
+                "id": "yaml",
+                "aliases": [
+                    "YAML",
+                    "yaml"
+                ],
+                "extensions": [
+                    ".yml",
+                    ".eyaml",
+                    ".eyml",
+                    ".yaml"
+                ],
+                "firstLine": "^#cloud-config"
```

Comment:
what does this mean?

Reply:
Initially, one of the ideas to limit linting scope was to use `firstLine`, but now since documentorSelector is solving the purpose it is not required. Will remove it.